commissioning.tex

```diff
@@ -1,27 +1,50 @@
 \section{Rubin Observatory Commissioning}
 \label{sec:commissioning}
 
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ''First Photon`` and ''System First Light``. 
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the LSSTCam focal plane to produce an image of the night sky.
+
+\textbf {System First Light}: Defined as the first science-grade images of the night sky obtained with LSSTCam. 
+Also referred to as \textbf{First Light}. 
+These images will be shared with the worldwide community. 
+
+First Photon occurs following the successful completion of system integration. 
+There are no quality criteria applied to the images produced for the first photon milestone. 
+First Photon  marks the start of the the on-sky engineering phase.
+System First Light  marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases of commissioning.
+The time from First Photon to System First Light is anticipated to be three months.
+During these three months, the focus will be on fine tuning the system including optical alignment and improving the image quality, collecting calibration data, and carrying out \textit{First Look} science programs. 
+
+For a detailed description of all the commissioning milestones and the most current dates, see \citeds{dmtn-232}
+
 \subsection{Commissioning Schedule}
 \label{ssec:commissioning-schedule}
 
-In December 2022, as a result of reoptimizing the sequence of integration activities, the Rubin Construction Project decided to install the LSST Science Camera (LSSTCam) on the telescope earlier in the assembly sequence than previously planned. 
-As a consequence, no on-sky data will be taken with the Commissioning Camera (ComCam). 
-This change in strategy will produce more early-commissioning data than would have been available with ComCam on a similar timeframe. 
+Following the October 2023 project schedule workshop, the Rubin Construction Project is reoptimizing the sequence of integration activities based on important updates to projected subcomponent availability.
+Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
```

Comment:
```suggestion
Given recent delays in the Camera ship date and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
```